Source/plrmsg.h

```diff
@@ -16,6 +16,7 @@ struct _plrmsg {
 	Uint32 time;
 	uint8_t player;
 	char str[144];
+	char name[24];

```

Comment:
A few things could probably be simplified by turning this in to two std::string instead of char arrays. Let me know if you would like me to apply the change?

Reply:
Absolutely

Reply:
Done